app/controllers/signs_controller.rb

```diff
@@ -4,11 +4,11 @@ class SignsController < ApplicationController
   layout 'admin', except: [:play]
 
   skip_before_action :authenticate_user!, only: [:play, :poll]
-  
+
   before_action :set_sign, only: [:remove_slide, :poll, :show, :edit, :update, :destroy, :play, :settings, :order]
   before_action :set_search_filters, only: [:index]
 
-  load_and_authorize_resource
+  load_and_authorize_resource except: [:play, :poll]
```

Comment:
huh. I can take a look at it sometime soon when I have a moment of downtime. I'll be in the signage app for a bit anyways for my current task.